internal/logprocessor/logprocessor.go

```diff
@@ -60,8 +61,12 @@ func (c *Client) parseResponse(body []byte) ([]byte, error) {
 	}
 
 	conn := connection{}
-
-	conn.SrcIP = rb.XFwdFor
+	if strings.Contains(rb.XFwdFor, ",") {
+		parsedIP := strings.Split(rb.XFwdFor, ",")
+		conn.SrcIP = parsedIP[0]
+	} else {
+		conn.SrcIP = rb.XFwdFor
```

Comment:
what happens here if rb.XFwdFor is empty? it wouldnt contain a `,` which would send it into its else. Which would overwrite conn.SrcIP, deleting the good original ip right?

Reply:
You're right, I need to do the validIP := net.ParseIP(rb.XFwdFor) first I think.  

Opinion on that?

Reply:
sounds logical

Reply:
Glad you saw that. Also, Check my post below.